Gopkg.lock

```diff
@@ -120,6 +120,12 @@
   packages = ["unix","windows"]
   revision = "fb4cac33e3196ff7f507ab9b2d2a44b0142f5b5a"
 
+[[projects]]
+  branch = "v2"
+  name = "gopkg.in/yaml.v2"
+  packages = ["."]
+  revision = "cd8b52f8269e0feb286dfeef29f8fe4d5b397e0b"
+
```

Comment:
we're just using it to format some output? that can be done without that much code with a few printfs, or even standard library templates (but even that may be overkill)